app/config/AppConfig.scala

```diff
@@ -273,4 +272,7 @@ class AppConfig @Inject() (
   val glcActs16n = loadConfig("files.codelists.glc.acts")
   val glcRoro16r = loadConfig("files.codelists.glc.roro")
   val glcGvms16s = loadConfig("files.codelists.glc.gvms")
+
+  val holderOfAuthorisationCodeFile = loadConfig("files.codelists.hoa.hoa-codes")
+  val taggedHolderOfAuthorisationCodeFile = loadConfig("files.codelists.hoa.tagged-hoa-codes")
```

Comment:
Could you perhaps rename the `hoa` folder to `holdersOfAuthorisation`. The 'hoa' abbreviation for the individual files should be ok as long as the folder has the full name. Otherwise our successors may scratch their head what hoa stands for :slightly_smiling_face: